src/main/groovy/com/lesfurets/jenkins/unit/declarative/AllOfDeclaration.groovy

```diff
@@ -26,6 +33,18 @@ class AllOfDeclaration extends WhenDeclaration {
         return true
     }
 
+    def anyOf(delegate) {
+        def exp_result;
+        for (def exp in this.anyOfs) {
+            exp_result = exp.execute(delegate)
+            //exp_result = execute(delegate, exp)
```

Comment:
Why is this line commented out? I suspect you meant to delete it. :slightly_smiling_face: